scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/ChangeDBConfiguration.java

```diff
@@ -0,0 +1,686 @@
+    private static CommandLine getCommandLine(String[] args, Options options) throws ChangeDBConfigurationException {
+        Option opt = new Option(HELP_OPTION, HELP_OPTION_NAME, false, "Display this help");
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option(DEBUG_OPTION, DEBUG_OPTION_NAME, false, "Debug mode (prints modified files and properties)");
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option(COMPONENT_OPTION,
+                         COMPONENT_OPTION_NAME,
+                         false,
+                         "Target component to configure (rm, scheduler, catalog, service-automation, notification or all to configure all components with a common configuration)");
+        opt.setRequired(false);
+        opt.setArgName(COMPONENT_OPTION_NAME.toUpperCase());
+        opt.setArgs(1);
+        options.addOption(opt);
+
+        opt = new Option(DB_VENDOR_OPTION,
+                         DB_VENDOR_OPTION_NAME,
+                         false,
+                         "Target database vendor to configure (hsqldb, postgresql, mysql, oracle)");
+        opt.setRequired(false);
+        opt.setArgName(DB_VENDOR_OPTION_NAME.toUpperCase());
+        opt.setArgs(1);
+        options.addOption(opt);
+
+        opt = new Option(URL_OPTION,
+                         URL_OPTION_NAME,
+                         false,
+                         "Database JDBC url that will be used to access the target component database. If not used, the url will be inferred from --" +
+                                DB_HOST_OPTION_NAME + " and --" + DB_PORT_OPTION_NAME +
+                                " options. Note: this parameter is the only way to use the TNS URL Format with Oracle database.");
+        opt.setRequired(false);
+        opt.setArgName(URL_OPTION_NAME.toUpperCase());
+        opt.setArgs(1);
+        options.addOption(opt);
+
+        opt = new Option(USERNAME_OPTION, USERNAME_OPTION_NAME, false, "Database user name");
+        opt.setRequired(false);
+        opt.setArgName(USERNAME_OPTION_NAME.toUpperCase());
+        opt.setArgs(1);
+        options.addOption(opt);
+
+        opt = new Option(PASSWORD_OPTION,
+                         PASSWORD_OPTION_NAME,
+                         false,
+                         "Database user password. Can be provided already encrypted using the 'encypt' command or in clear text. " +
+                                "Password will always be encrypted when stored in configuration files.");
+        opt.setRequired(false);
+        opt.setArgName(PASSWORD_OPTION_NAME.toUpperCase());
+        opt.setArgs(1);
+        options.addOption(opt);
+
+        opt = new Option(DB_HOST_OPTION,
+                         DB_HOST_OPTION_NAME,
+                         true,
+                         "Database hostname used (localhost, myserver) or IP address (127.0.0.1, 192.168.12.1). Cannot be used in conjunction with --" +
```

Comment:
add e.g., to (127.0.0.1, 192.168.12.1)
(e.g., 127.0.0.1, 192.168.12.1)

Reply:
done